src/github/nag.rs

```diff
@@ -395,7 +395,7 @@ fn evaluate_pendings() -> DashResult<()> {
                         proposal.id, why));
         }
 
-        let majority_complete = num_outstanding_reviews < num_complete_reviews;
+        let majority_complete = num_outstanding_reviews < 2 * num_complete_reviews;
```

Comment:
FWIW I think `<=` is more appropriate since 2/3 is already strictly more than 1/2. Your currently logic will consider 4/6 insufficient, I think.